### PR TITLE
Draft doc/UX for admin policy with domain names

### PR DESCRIPTION
**Pull request by neiljerram**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit c63a131807da739fd910954f9f52be1aedb10d8e

https://deploy-preview-2423--calico.netlify.com

File: _data/master/navbars/security.yml [hunk 1]
```diff
@@ -44,3 +44,5 @@ toc:
     path: /security/comms/reduce-nodes
   - title: Securing Calico's Prometheus endpoints
     path: /security/comms/secure-metrics
+- title: Admin policy using domain names
```
Comment (lxpollitt) [inline _data/master/navbars/security.yml]: I suggest "Use domain names in policy rules" as the title.  It's a little more "How too..." if you see what I mean.

File: master/security/domain-based-policy.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+2.  By creating a
+    [GlobalDomainSet]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/globaldomainset)
+    resource - which groups together a set of domain names that you
+    want to label in the same way - and using `domains.selector` to
+    select GlobalDomainSets by their labels.  The effect then is to
+    allow egress to all of the domain names listed in all of the
+    selected GlobalDomainSets.
+
+### Understanding which domain names will work
```
Comment (lxpollitt) [inline master/security/domain-based-policy.md]: I would like to review this section after we have got answers to the requirements questions from PM and after we've agreed the design we're going to implement.

File: _data/master/navbars/security.yml (hunk 1 above)
Comment (neiljerram) [reply _data/master/navbars/security.yml]: How to is good, but I think the title also needs to make clear that this is admin-only.  Would "Use domain names in admin policy rules" work for you?

File: master/security/domain-based-policy.md (hunk 2 above)
Comment (neiljerram) [reply master/security/domain-based-policy.md]: More precisely, I agree with reviewing after we're sure about requirements.  I think doc/spec/UX review should be before or at the same time as design review, not _after_ design review, surely?

Comment (lxpollitt) [reply master/security/domain-based-policy.md]: Yes, at the same time as design is more precise.  Agree!

File: _data/master/navbars/security.yml (hunk 1 above)
Comment (lxpollitt) [reply _data/master/navbars/security.yml]: "admin" means different things to different personas. e.g. if you were to ask someone what the difference between an "admin policy rule" and a "policy rule" is, I think you would get different answers from different personas.  One of the aims with the how-to titles is to be persona neutral.

**Pull request closed without merging**